### PR TITLE
Initial commit Shodan Solution.

### PR DESCRIPTION
**Pull request by rahul0216**
Change(s):
   - Added azuredeply.json files for custom connector and all playbooks
   - Added readme.md files for custom connector and all playbooks
   - Added images for custom connector and all playbooks

   Reason for Change(s):
   - Initial commit

   Version Updated:
   - Initial version 1.0.0

   Testing Completed:
   - End to end deployment testing
   - End to end functional testing 

-----------------------------------------------------------------------------------------------------------

### CODE REVIEW

**Comment by v-mchatla** [comment]
Hi Manish,
Can you please review and shared the feedback.
Thanks

File: Solutions/Shodan/Playbooks/CustomConnector/ShodanCustomConnector/azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,854 @@
...
+            }
+        }
+    },
+    "variables": {},
+    "resources": [
+        {
+            "type": "Microsoft.Web/customApis",
+            "apiVersion": "2016-06-01",
+            "name": "[parameters('ConnectorName')]",
+            "location": "centralindia",
```
Comment (manishkumar1991) [inline Solutions/Shodan/Playbooks/CustomConnector/ShodanCustomConnector/azuredeploy.json:26]: Hard coded location

File: Solutions/Shodan/Playbooks/CustomConnector/ShodanCustomConnector/readme.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+### Prerequisites to deploy Custom Connector 
+- Have Shodan API Endpoint Url handy (Default: https://api.shodan.io)
+
+### Deployment Instructions
+
+1. To deploy Custom Connector, click the Deploy to Azure button. This will launch the ARM Template deployment wizard.
+2. Fill in the required parameters:
+    - Connector Name: Enter the custom connector name (Default: ShodanCustomConnector)
+    - Service Endpoint: Enter the Shodan API endpoint (Default: https://api.shodan.io). Make sure to prefix with https://.
```
Comment (manishkumar1991) [inline Solutions/Shodan/Playbooks/CustomConnector/ShodanCustomConnector/readme.md:17]: In custom connector https:// and http:// is getting replaced by " "   then why asking customer to prefix with https://

"host": "[replace(replace(parameters('ServiceEndpoint'),'https://',''),'http://','')]",

**Comment by manishkumar1991** [comment]
"Deploy Azure" Button links needs to be corrected.

**Comment by rahul0216** [comment]
> "Deploy Azure" Button links needs to be corrected.

I'll update the button links once PR is merged to master branch.

**Review by manishkumar1991 — APPROVED** [review]
Approved

**Pull request merged**